Kinvey/Kinvey/Query.swift

```diff
@@ -70,19 +77,34 @@ public final class Query: NSObject, BuilderType, Mappable {
             }
             return NSExpression(forKeyPath: keyPath)
         case .constantValue:
-            if otherSideExpression.expressionType == .keyPath,
-                let (_, optionalTransform) = persistableType?.propertyMapping(otherSideExpression.keyPath),
-                let transform = optionalTransform
-            {
-                return NSExpression(forConstantValue: transform.transformToJSON(expression.constantValue))
-            }
+            #if !os(watchOS)
```

Comment:
I'm guessing you had to add this for some compilation error on watchOS. Just out of curiosity, what *should* happen in this code if run on watchOS? Can you add some default behavior or error message that would make sense?